app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+class User < ActiveRecord::Base
+  has_many :follows_followers, class_name: 'Follow', foreign_key: :following_id
+  has_many :follows_following, class_name: 'Follow', foreign_key: :follower_id
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.